Archisinal.md

```diff
@@ -0,0 +1,310 @@
+## Archisinal - NFT Marketplace ##
+
+- **Team Name:** Archisinal Technologies
+- **Payment Address:** 14xeAxD7LSJQmNXn2RQcm6WTJiByv6HKn6T8fTjqX1HeCMwR
+
+## Project Overview :page_facing_up:
+### Overview
+
+- Archisinal wants to become the Opensea for professionals in Architecture, Engineering and Construction (AEC).
```

Comment:
This is a very unique and intriguing idea. As a former architect, I support this :)

Reply:
This is great news, Sam! We should have a call - always looking for feedback from different industry perspectives.